Documentation/DSCResources/VMHostVirtualPortGroup/VMHostVirtualPortGroup.md

```diff
@@ -0,0 +1,115 @@
+# VMHostVirtualPortGroup
+
+## Parameters
+
+| Parameter | Attribute | DataType | Description | Allowed Values |
+| --- | --- | --- | --- | --- |
+| **Server** | Key | string | Name of the Server we are trying to connect to. The Server can only be a vCenter. ||
+| **Name** | Key | string | Name of the VMHost to configure. ||
+| **Credential** | Mandatory | PSCredential | Credentials needed for connection to the specified Server. ||
+| **PortGroupName** | Key | string | The name for the Port Group. ||
+| **VirtualSwitch** | Mandatory | string | The Virtual Switch associated with the Port Group. The Virtual Switch must be a Standard Virtual Switch. ||
+| **Ensure** | Mandatory | Ensure | Value indicating if the Port Group should be Present or Absent. | Present, Absent |
+| **VLanId** | Optional | int | The VLAN ID for ports using this Port Group. | 0 - 4095 |
+
+## Description
+The resource is used to create, update and delete Port Groups which are associated with the specified Virtual Switch.
```

Comment:
Describe what happens when Absent is given on the Ensure property when there are VMs connected to the PortGroup here in the documentation.

Reply:
Done.